### PR TITLE
SAM-2803: Shading and border inconsistencies creating Assessment/Ques…

### PR DESCRIPTION
**Pull request by ramonchu2k**
…tion pool using mark-up text

### CODE REVIEW

**Comment by ottenhoff** [comment]
Can you summarize the code change?

**Comment by ramonchu2k** [comment]
Sure, the changes are:
1. Surround with <div class="table-responsive"> the dataTable at line 58.
2. Change the styleclasses at line 59.
3. Add some "column-list" at columnClasses at line 60.
4. Remove a panelGrid and its panelGroups that were surround the dataTables in lines: 98,112,127 y 142 and set the rendered option from panelGroups into these dataTables.

**Comment by ottenhoff** [comment]
Okay looks good! Thanks.
